CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Contributing
+
+## Data Migrations
+
+We have a few tips you should follow when writing data migrations:
+
+- There's no need to represent the the editions within the migration, instead
+  you can run the `represent_downstream:` class of rake tasks through Jenkins.
+  This prevents the need to have a check for production since the Sidekiq
+  workers are not available in the other environments, and means you do not
+  have to use `disable_ddl_transaction!`.
+- If the record you are trying to modify doesn't exist, simply move on to the
+  next one rather than failing. This prevents new clones of the
+  `publishing-api` from having to replicate their data from production before
+  being able to run the migrations.
+
+We also recommend that every so often, old data migrations are deleted since
+they are most likely no longer useful.
```

Comment:
The content here looks good but I think it maybe implies quite a high understanding.

I'd maybe try going more along this sort of lines:

## Are you writing a migration to change Publishing API data?

We have a lot of migrations added to the Publishing API to change data, our long term plan is to reduce these so that these can be done via the API itself and not need migrations. Until we are at that point please follow the following guidelines when creating migrations.

### Firstly run the migration first on your machine first
Why
Include details in PR of how long it took against reasonably fresh set of data

### Always include the schema.rb file
Even if it's just the timestamp
Why

### Target only content related to the application you are affecting
e.g include where "publishing_app: whitehall" to reduce risk of affecting another app, even if it seems unlikely

### Don't assume the records you are altering exist
Migrations are run in test environments, on old dbs etc

### Do you need to represent this data downstream?
This will break local machines
Use rake task ideally separate
if not disable transaction and use


### If you are disabling transaction ensure the migration is idempotent
Why

What do you think?

As a side idea - rather than using the if env == production we could consider having an ENV var for checking whether you can represent_downstream